src/Features/CSharp/Portable/InitializeParameter/InitializeParameterHelpers.cs

```diff
@@ -71,6 +73,15 @@ public static bool IsImplicitConversion(Compilation compilation, ITypeSymbol sou
                     return;
                 }
 
+                if (convertedStatement is ReturnStatementSyntax convertedReturn &&
+                    convertedReturn.Expression is not null &&
+                    body is ArrowExpressionClauseSyntax arrowClause &&
+                    arrowClause.ArrowToken.TrailingTrivia.IndexOf(SyntaxKind.EndOfLineTrivia) == -1)
+                {
+                    var whiteSpaceTrivia = convertedReturn.Expression.DescendantTrivia().Where(static tr => tr.IsWhitespace() && !tr.IsElastic());
+                    convertedStatement = convertedStatement.ReplaceTrivia(whiteSpaceTrivia, (_, tr) => tr.AsElastic());
```

Comment:
interesting idea.  we haven't really done this elsewhere.  but i have no issues with it.